plugins/object-store-plugin/src/test/groovy/testhelpers/MinioContainer.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Rundeck, Inc. (http://rundeck.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testhelpers
+
+import io.minio.MinioClient
+import org.testcontainers.containers.GenericContainer
+
+class MinioContainer<SELF extends MinioContainer<SELF>> extends GenericContainer<SELF> {
```

Comment:
FYI you don't **have** to use the self typing here, and can define it as `MinioContainer extends GenericContainer< MinioContainer>`